client/src/components/pages/home/case-study.tsx

```diff
@@ -1,7 +1,7 @@
 import { useIntl, FormattedMessage } from 'react-intl';
```

Comment:
```suggestion
```

We aren't using this any more so we can remove it. This is causing build errors in the test action on this PR